<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>File Indexer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/buttons/resources/search.png</normaloff>:/buttons/resources/search.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionSearch"/>
  </widget>
  <action name="actionStart">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/buttons/resources/play1normal.png</normaloff>
     <normalon>:/buttons/resources/play1disabled.png</normalon>:/buttons/resources/play1normal.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/buttons/resources/pausepressed.png</normaloff>:/buttons/resources/pausepressed.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/buttons/resources/stop1normalorange.png</normaloff>:/buttons/resources/stop1normalorange.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/buttons/resources/search.png</normaloff>:/buttons/resources/search.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="toolTip">
    <string>Search</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
